common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -1138,4 +1135,33 @@ public void finalizeBeforeSendingResult(@NonNull final RawAuthorizationResult re
         ((AbstractSmartcardCertBasedAuthChallengeHandler<?>)mCertBasedAuthChallengeHandler).promptSmartcardRemovalForResult(callback);
     }
 
+    /**
+     * Create a span with parent span context if available.
+     * @param spanName Name of the span to be created.
+     * @return Created {@link Span}
+     */
+    private Span createSpanWithAttributesFromParent(@NonNull final String spanName) {
```

Comment:
Should we have this in SpanExtension instead, or some central place to access in other places? This seems generally useful